pilot/cmd/pilot-agent/status/server.go

```diff
@@ -149,67 +96,32 @@ func (s *Server) Run(ctx context.Context) {
 	// Wait for the agent to be shut down.
 	<-ctx.Done()
 	log.Info("Status server has successfully terminated")
+	s.notifyReady()
```

Comment:
Not clear why this is here?

Reply:
I think an artifact of some earlier versions of this code. removed.